### PR TITLE
add pytorch inferenceModel

### PR DESCRIPTION
**Pull request by glorysdj**
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/1765)

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/ModelLoader.scala [hunk 1]
```diff
@@ -76,5 +76,15 @@ object ModelLoader extends InferenceSupportive {
       model
     }
   }
+
+  def loadFloatModelForPyTorch(modelPath: String)
+  : AbstractModule[Activity, Activity, Float] = {
+    timing("load model") {
+      logger.info(s"load model from $modelPath")
```
Comment (qiyuangong) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/ModelLoader.scala]: I think these 2 log  can be merged into 1. :)

**Comment by qiuxin2012** [comment]
Please add a unit test

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/ModelLoader.scala (hunk 1 above)
Comment (glorysdj) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/ModelLoader.scala]: sure

**Comment by glorysdj** [comment]
ut added

**Comment by glorysdj** [comment]
http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/2760/

**Pull request merged**